sync/bundle/message/hello_test.go

```diff
@@ -45,6 +46,26 @@ func TestHelloMessage(t *testing.T) {
 		assert.Equal(t, errors.Code(m.BasicCheck()), errors.ErrInvalidPublicKey)
 	})
 
+	t.Run("MyTimeUnixMilli of time1 is less or equal than hello message time", func(t *testing.T) {
+		time1 := time.Now()
+		myTimeUnixMilli := time1.UnixMilli()
+
+		m := NewHelloMessage(ts.RandPeerID(), "Alice", 100, 0, ts.RandHash(), ts.RandHash())
+
+		assert.LessOrEqual(t, m.MyTimeUnixMilli, time.Now().UnixMilli())
+		assert.GreaterOrEqual(t, m.MyTimeUnixMilli, myTimeUnixMilli)
+	})
+
+	t.Run("MyTimeUnixMilli of time1 is greater than hello message time", func(t *testing.T) {
```

Comment:
This test is redundant. right? We almost cover every thing in above test.

Reply:
@b00f 
yes, I've forgot to remove this test, I'll fix it right now